packages/dds/tree/src/feature-libraries/defaultFieldKinds.ts

```diff
@@ -297,13 +293,16 @@ const valueRebaser: FieldChangeRebaser<ValueChangeset> = isolatedFieldChangeReba
 	invert: (
 		{ revision, change }: TaggedChange<ValueChangeset>,
 		invertChild: NodeChangeInverter,
+		reviver: NodeReviver,
 	): ValueChangeset => {
 		const inverse: ValueChangeset = {};
 		if (change.changes !== undefined) {
-			inverse.changes = invertChild(change.changes);
+			inverse.changes = invertChild(change.changes, 0);
 		}
-		if (change.value !== undefined) {
-			inverse.value = { revert: revision };
+		if (revision !== undefined && change.value !== undefined) {
+			// TODO
+			// assert(revision !== undefined, 0x478 /* Unable to revert to undefined revision */);
```

Comment:
Does this assert break our ability to complete a transaction?

Reply:
Looks like you already addressed what I thought might have been the reason for this assert to be a problem so I'm curious if there is some other reason why I can't be enabled.

Reply:
Nope, I just forgot about this :)